addons/account/tests/test_invoice_tax_totals.py

```diff
@@ -845,6 +845,39 @@ def test_recompute_cash_rounding_lines(self):
         for move in moves_rounding:
             self.assertEqual(sum(move.line_ids.filtered(lambda line: line.display_type == 'rounding').mapped('balance')), moves_rounding[move])
 
+    def test_recompute_cash_rounding_lines_multi_company(self):
```

Comment:
Even if we don't get the traceback, I'll personally add an assert to make sure that we got the correct account for the cash rounding line. nothing protect me and ensure me that we didn't got a silent error here